pkg/minikube/cluster/cluster.go

```diff
@@ -437,7 +440,13 @@ func GetVMHostIP(host *host.Host) (net.IP, error) {
 		}
 		return ip, nil
 	case "virtualbox":
-		ip, err := getIPForInterface("vboxnet0")
+		out, err := exec.Command("vboxmanage", "showvminfo", "minikube", "--machinereadable").Output()
```

Comment:
The docker machine driver does an os-specific lookup for the VboxManage binary.  Do we need to do that here?  Also, I think the binary is usually named  VboxManage, so we might have to explicitly call that with the full path.

https://github.com/docker/machine/blob/c4c3f5f9e2d3957050b7645cc9fccd98f78b2241/drivers/virtualbox/virtualbox.go#L1011